docker-wrappers/SPRAS/README.md

```diff
@@ -0,0 +1,54 @@
+# SPRAS Docker image
+
+## Building
+A Docker image for SPRAS that is available on [Dockerhub]()
```

Comment:
```suggestion
A Docker image for SPRAS that is available on [DockerHub](https://hub.docker.com/repository/docker/reedcompbio/spras)
```

I built and pushed the image with the `latest` tag. I'll push the versioned verion once we decide what versioning scheme to use.